matRadGUI.m

```diff
@@ -115,6 +115,16 @@ function matRadGUI_OpeningFcn(hObject, ~, handles, varargin)
 axis equal off;
 set(f, 'AlphaData', alpha);
 
+% turn off the datacursormode (since it does not allow to set scrolling
+% callbacks
+handles.dcm_obj = datacursormode(handles.figure1);
+set(handles.dcm_obj,'DisplayStyle','window');
+if strcmpi(handles.dcm_obj.Enable,'on')
```

Comment:
I changed all property accesses from struct to get/set access... Didn't really know this.. is this only the case for objects with properties or generally for structs in older MATLAB versions?